src/Microsoft.AspNetCore.Server.Kestrel/Internal/Exceptions/BadHttpRequestException.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.IO;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Exceptions
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Exceptions
 {
     public sealed class BadHttpRequestException : IOException
```

Comment:
I believe this can be surfaced to the caller via `Request.Body.Read(Async)` for invalid chunking in the request body, so this shouldn't be in an internal namespace. @cesarbs